pkg/yurttunnel/iptables/iptables.go

```diff
@@ -157,13 +159,54 @@ func (im *iptablesManager) Run(stopCh <-chan struct{}) {
 		select {
 		case <-stopCh:
 			klog.Info("stop the iptablesManager")
+			im.cleanupIptableSetting()
 			return
 		case <-ticker.C:
 			im.syncIptableSetting()
 		}
 	}
 }
 
+func (im *iptablesManager) cleanupIptableSetting() {
+	dnatPorts, err := util.GetConfiguredDnatPorts(im.kubeClient, im.insecurePort)
```

Comment:
It's not need to get the current dnat ports, only use `im.lastDnatPorts` to cleanup iptables instead.

Reply:
ok